framework/doc/content/sqa/minimum_requirements.md

```diff
@@ -6,7 +6,10 @@ In general, the following is required for MOOSE-based development:
 
   - Note: Intel compilers are not supported.
 
-- Memory: 16 GBs (debug builds)
-- Processor: 64-bit x86
-- Disk: 30GB
-- Many flavors of modern Linux and the two most recent versions of MacOS
+- Memory: 8 GBs of RAM for optimized compilation (16 GBs for debug compilation), 2 GB per core execution
+- Processor: 64-bit x86 or ARM64 (specifically, Apple Silicon)
+- Disk: 100GB
```

Comment:
Would welcome some comment here on minimum disk usage - we had 30GB in one place and 100GB in another....

Reply:
I had the same thought when I saw the descrepancy.....there's no way in heck this is the "minimum"....I kept it in to prompt a discussion.

Reply:
After building the main three methods (`opt`,`devel`, and `dbg`), the size of the MOOSE directory was about 21GB, so I think I will stick with 30GB here. Updated in the most recent commit.

Reply:
did you build combined in all 3? It's excessive to me

Reply:
I don't agree that 30 is excessive. By building all three combined executables, @friedmud and I wanted to create a worst case scenario where one might want multiple copies of the complete executable in place with comfortable space for simulations, analysis, and debugging. Derek suggested adding 10GB to this, while I felt 9 (to get to a round number) was acceptable. 30GB happens to also be the min drive space we've suggested to people for years, with 100GB being the anomaly, so I'm comfortable with this.